drafts/2020-02-20-Rest-client-generator.adoc

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "The OpenAPI Generator has added support for Eclipse MicroProfile"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/u/28316667
+author_github: https://github.com/turkeylurkey
+seo-title: OpenAPI Generator has added support for Eclipse MicroProfile - OpenLiberty.io
+seo-description: Develop microservices quicker using the OpenAPI Generator and the Eclipse MicroProfile programming framework.
+blog_description: "Develop microservices quicker using the OpenAPI Generator and the Eclipse MicroProfile programming framework."
+---
+= Generate MicroProfile REST client code to send requests to other microservices
+Paul Gooderham <https://github.com/turkeylurkey>
+
+Sending a request to another microservice from a MicroProfile app has meant manually writing the REST client
+code in the app but now you can use the OpenAPI Generator to generate the REST client code for you.
+If the API of the microservice to which the request is sent is documented using OpenAPI,
+the developer would usually manually write the client code based on that documentation.
+Now, however, the developer can just run the OpenAPI Generator against the OpenAPI documentation
+for the API and quickly generate the framework for the REST client code.
+
+The REST client in the MicroProfile app is a collection of Java interfaces and optionally implementation
+methods to access another microservice. Check out the educational guide
+https://openliberty.io/guides/microprofile-rest-client.html[Consuming RESTful Java microservices].
+Pay attention to the sections "Injecting the client with dependency injection" and
```

Comment:
Instead of `Pay attention`, describe why the user should read those sections e.g. `The "Injecting the client with dependency injection" and "Building the client with RestClientBuilder" sections describe how a REST client can be used in an application.`